exercism/bob/bob.js

```diff
@@ -0,0 +1,14 @@
+var Bob = function() {};
+
+Bob.prototype.hey = function(text) {
+  if (text.match(/[A-Z]/) && text === text.toUpperCase()) {
```

Comment:
I know it feels redundant however if you remove part of the if statement, then various tests fail (the tests are really mean! haha). I agree with you there is probably a way to re-factor this and make it better. At this point it was the only way we could get all tests passing